src/core/MOM.F90

```diff
@@ -191,7 +200,8 @@ module MOM
                                      !! set by calling the function useRegridding() from the
                                      !! MOM_regridding module.
   logical :: do_dynamics             !< If false, does not call step_MOM_dyn_*. This is an
-                                     !! undocumented run-time flag that is fragile.
+  logical :: do_online               !< If false, does not call step_MOM_dyn_*. This is an
```

Comment:
I just pushed a commit that deletes that extra line. step_MOM.F90. No other references were found in step_MOM. Do you think I should clarify in the parameter description of DO_ONLINE that the flag is only used in the solo driver?